pyuvdata/uvdata/ms.py

```diff
@@ -576,7 +588,7 @@ def _write_ms_pointing(self, filepath):
         # per-timestamp, per-antenna entry
         times = np.asarray(
             [
-                Time(t, format="jd", scale="utc").tai.mjd * 3600.0 * 24.0
+                Time(t, format="jd", scale="utc").mjd * 3600.0 * 24.0
```

Comment:
`Time().mjd.to(u.s).value` or `86400.0` for consistency.

Reply:
Changed to `86400.0` in 5c36be1a.